tools/test/examples/README.md

```diff
@@ -0,0 +1,81 @@
+# Examples testing script
+
+examples.py is script which makes easier to test examples.
```

Comment:
Suggestion; it might be useful to have more description here of what this program actually does. This is pretty brief. Are there any preconditions? What directory structure does the script expect? Are there any external dependencies (eg toolchain installations) and, if so where should they be configured. Where do the results go, and what form do those results take?

Reply:
I know that there is room for improvements. This is the beginning of documenting this feature

Reply:
ok